source/js/components/ExpandedRecipeCard.js

```diff
@@ -1,238 +1,751 @@
-// eslint-disable-next-line no-unused-vars
-class ExpandedRecipe extends HTMLElement {
-  constructor() {
-    super();
-    this.attachShadow({ mode: 'open' });
-
-    // Need to determine format of expanded recipe card
-
-    const styles = document.createElement('style'); // for CSS styles
-    const article = document.createEleement('article'); // for expanded recipe card
-
-    // ***************
-    // Need to determine format of CSS for recipe card
-    // ** REPLACE THE BELOW (TAKEN FROM LAB 7, BUT NOT ALLOWED TO BE USED;
-    // ** IT'S ONLY HERE TO TEST)
-    styles.innerHTML = `
-    article {
-      background-color: white;
-      box-shadow: 0 0 10px rgb(0 0 0 / 15%);
-      margin: 30px auto;
-      max-width: 720px;
-      padding: 25px;
-      transition: all 0.2s ease;
-      width: 80%;
+import getRouter from '/source/js/main.js';
+
+// RecipeExpand.js
+class RecipeExpand extends HTMLElement {
+    constructor() {
+      super();
+      this.attachShadow({ mode: 'open' });
+  
+      // Create styles and root element
+      const styles = document.createElement('style');
+
+      // Fill in styles and root element
+      styles.innerHTML = `
+      @import url("https://fonts.googleapis.com/css2?family=Lato&display=swap");
+        
+      html,
```

Comment:
the styles here should match antonia and jon's design exactly from their example code in `source/demo-code/recipe-info-rough/recipe-page.css`. we should just copy all of the css from that file into the style element here.